modules/installation-and-upgrade/pages/container-deployment/suma/server-migration-suma.adoc

```diff
@@ -82,6 +82,13 @@ By matching or exceeding these specifications, you can help ensure a smooth and
 mgradm migrate podman <source.fqdn>
 ----
 
+[IMPORTANT]
+====
+
+After successfully running the [command]``mgradm migrate` command, the Salt setup on all minions will still point to the old server. To redirect them to the new server, it is required to use the same FQDN and IP address for the new server as the old one.
```

Comment:
```suggestion
After successfully running the [command]``mgradm migrate`` command, the {salt} setup on all clients will still point to the old server (source server).
To redirect them to the new server (destination server), it is required to use the same FQDN and IP address for the new server as the old one.
```

Reply:
what is a source server ?

important is that it's the 4.3 server

Reply:
same domain name